index.html

```diff
@@ -1028,6 +1061,461 @@ <h2>
         </dd>
       </dl>
     </section>
+    <section data-dfn-for="GamepadHapticActuator" data-link-for="GamepadHapticActuator">
+      <h2>
+        <dfn>GamepadHapticActuator</dfn> Interface
+      </h2>
+      <p>
+        A {{GamepadHapticActuator}} corresponds to a configuration of motors or
+        other actuators that can apply a force for the purposes of haptic
+        feedback.
+      </p>
+      <pre class="idl">
+        [Exposed=Window]
+        interface GamepadHapticActuator {
+          readonly attribute FrozenArray&lt;GamepadHapticEffectType&gt; effects;
+          Promise&lt;GamepadHapticsResult&gt; playEffect(
+              GamepadHapticEffectType type,
+              optional GamepadEffectParameters params = {});
+          Promise&lt;GamepadHapticsResult&gt; reset();
+        };
+      </pre>
+      <p>
+        Instances of {{GamepadHapticActuator}} are created with the internal
+        slots described in the following table:
+      </p>
+      <table class="simple">
+        <tr>
+          <th>
+            Internal slot
+          </th>
+          <th>
+            Initial value
+          </th>
+          <th>
+            Description (non-normative)
+          </th>
+        </tr>
+        <tr>
+          <td>
+            <dfn data-dfn-for="GamepadHapticActuator">[[\effects]]</dfn>
+          </td>
+          <td>
+            An empty [=list=].
+          </td>
+          <td>
+            A [=list=] of {{Gamepad/GamepadHapticEffectType}} values
+            representing all the effects supported by the actuator
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <dfn data-dfn-for=
+            "GamepadHapticActuator">[[\playingEffectPromise]]</dfn>
+          </td>
+          <td>
+            `undefined`
```

Comment:
I wonder if we should use `null` instead of `undefined` for this. Seems like a better fit.

Reply:
I made the change, but could you please explain to me why `null` is a better fit?